apps/communications/dialer/test/unit/dialer_test.js

```diff
@@ -826,44 +829,4 @@ suite('navigation bar', function() {
       sinon.assert.called(NavbarManager.show);
     });
   });
-
```

Comment:
Right, this is a leftover of my first patch, I must have included it erroneously when commit --amend'ing the patch. I'll restore it.